### PR TITLE
Add an start_assisted_install step

### PR DESCRIPTION
**Pull request by flaper87**
This step starts the assisted agent unit so that the rest of the install
process can be managed by AIA.

Signed-off-by: Flavio Percoco <flavio@redhat.com>

### CODE REVIEW

File: hardware_manager/ironic_coreos_install.py [hunk 1]
```diff
@@ -13,8 +13,12 @@
 import json
 import os
 import subprocess
+import time
+
+import dbus
```
Comment (dtantsur) [inline hardware_manager/ironic_coreos_install.py:18]: I suspect we need to add python-dbus (or how it's called) to the package list

Comment (elfosardo) [reply hardware_manager/ironic_coreos_install.py:18]: yes, we need that, the name should be python3-dbus

Comment (flaper87) [reply hardware_manager/ironic_coreos_install.py:18]: Indeed! Although, I think it's already part of the image, which is why I didn't have to add it to setup.cfg during my tests.

I will update the file once I clean this PR.

**Pull request merged**